020-LinuxFundamentals/README.md

```diff
@@ -0,0 +1,95 @@
+	 
+## Prerequisites
+- Your own Azure subscription with contributor access for the Challenge 01 or contributor access to a pre-created Azure Virtual Machine for all other challenges.
+- Access to a terminal. The terms “terminal,” “shell,” and “command line interface” are often used interchangeably, but there are subtle differences between them:
+
+	* A terminal is an input and output environment that presents a text-only window running a shell.
+	* A shell is a program that exposes the computer’s operating system to a user or program. In Linux systems, the shell presented in a terminal is a command line interpreter.
+	* A command line interface is a user interface (managed by a command line interpreter program) which processes commands to a computer program and outputs the results.
+When someone refers to one of these three terms in the context of Linux, they generally mean a terminal environment where you can run commands and see the results printed out to the terminal.
+
+		Becoming a Linux expert requires you to be comfortable with using a terminal. Any administrative task, including file manipulation, package installation, and user management, can be accomplished through the terminal. The terminal is interactive: you specify commands to run and the terminal outputs the results of those commands. To execute any command, you type it into the prompt and press ENTER.
+
+		For our activites, it is recommended to use the [Azure Cloud Shell](http://shell.azure.com/).
+
+
+- There are some basic concepts that will be nice if you have them. If you would like to take a look they are [listed here](./Student/resources/concepts.md).
+- Concepts around the Linux Filesystem Hierarchy Standard (FHS) are recommended, so you can get more details about it [here](./Student/resources/fhs.md).
+- Regarding Linux commands, just for reference, [here is](./Student/resources/commands.md) a good cheat sheet.
+- The Linux man pages are your best friend. Make sure to use as best you can.
+
+## Learning Resources
+
+* [Linux Journey](https://linuxjourney.com/)
+* [Linux Upskill Challenge](https://linuxupskillchallenge.org/)
+* [Beginner's Guide for Linux - Tecmint](https://www.tecmint.com/free-online-linux-learning-guide-for-beginners/)
+* [Preparation for Linux Foundation Certified System Adminsitrator](https://github.com/Bes0n/LFCS)
+* [Linux Foundation Certified System Administrator (LFCS) Notes](https://github.com/simonesavi/lfcs)
+* [The Linux Documentation Project](https://tldp.org/)
+* [Intrduction to Linux - from TLPD](https://tldp.org/LDP/intro-linux/intro-linux.pdf)
+* [Linux Commands Notes for Professionals](https://goalkicker.com/LinuxBook/LinuxNotesForProfessionals.pdf)
+* [Introduction to Linux - Free course on Linux Foundation](https://training.linuxfoundation.org/training/introduction-to-linux/)
+
+## Feedback
+Please [share](https://forms.office.com/r/1W73Y1rrxu) your experience during this Linux Hackathon and help us to improve.
```

Comment:
Is this form available publicly?  Or only to MS employees?  If it is only available to MS employees, please remove it as this is a public repo/website.  Instead consider sending them to our WTH Feedback form at this link: 
https://github.com/microsoft/WhatTheHack/issues/new?assignees=&labels=suggestion&template=suggestion.yml&title=%5BSuggestion%5D%3A+

Reply:
This form is available publicly